pkg/i18n/english.go

```diff
@@ -770,6 +770,9 @@ type TranslationSet struct {
 	Actions                              Actions
 	Bisect                               Bisect
 	Log                                  Log
+	BreakingChangesTitle                 string
+	BreakingChangesMessage               string
+	BreakingChangesByVersion             map[string]string
```

Comment:
I changed my mind with regard to using reflection. When we serialize to JSON, a map will actually look identical to members, so it won't make a difference for Crowdin. And the map is easier for us, so I went with that.

Reply:
The map solution looks quite ergonomic in the actual code. I think this is a better solution to the direction I was going based on the initial issue discussion. I also like that it's now explicit that this is about breaking changes rather than "release notes highlights".